2019/Narek_Harutyunyan/Singleton.cpp

```diff
@@ -1,25 +1,34 @@
 //Singleton Design Pattern
 #include <iostream>
 #include <string>
+#include <mutex>   
 
 using namespace std;
```

Comment:
Never use entire namespace. Add using statements for specific objects or write std:: prefix for all usages.